extensions/extensions-helm/extensions-helm-spark-infrastructure/aissemble-spark-history-chart/Chart.yaml

```diff
@@ -5,7 +5,7 @@ version: 1.0.0
 apiVersion: v2
 description: Baseline Spark History chart -- see the extensions-helm directory in aiSSEMBLE for more details.
 # App Version is automatically set by the plugin so this is only a placeholder
-appVersion: 1.0.0
+appVersion: 1.7.0-SNAPSHOT
```

Comment:
per the comment, doesn't this get updated automatically? I thought the other charts also just have appVersion 1.0.0

Reply:
Good catch, this is left over from local testing.  Fixing now.